### PR TITLE
Proofing your index changes

### PR DESCRIPTION
**Pull request by jjon-saxton**
Thinking we should do this through pull request, maybe. Anyway, your gallery indices seemed fine, but I have a few suggestions in your introduction for you to review. I didn't change them directly, so maybe accept the pull request and then actually apply the changes.

### CODE REVIEW

File: content/en/introduction.org [hunk 1]
```diff
@@ -10,12 +10,12 @@
 
 Hewwo hewwo nyaoo nyaoo~~
 
-My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent to protect, my kittens!
+My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent +to protect+ (here for, "protect" seems redundant here), my kittens!
```
Comment (yayoi-chi) [inline content/en/introduction.org:13]: I think you're right, "protect" does seem redundant. "I was sent here for" doesn't quite seem like something i'd say though. How about "I have had to find new ways to communicate in order to fulfill my mission (or task)"?

File: content/en/introduction.org [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
 
 You can get recaps, and updates on this website, which the kind folks taking care of me in your world have made for me, but you can also get updates on [[https://mastodon.world/@yaya][Mastodon]] and my [[https://www.youtube.com/@yayoi-chi/community][YouTube Community Tab]]! I love helping and have also created some [[{{% ref guides %}}][Guides]] and [[{{% ref reviews %}}][Reviews]] of some of my favorite games for you!
 
 My mission above all is to be a great friend! So please help me out and connect with me. I might not be able to talk anymore, but I can still communicate and I love getting to know new people. Also, if you would like to become one of my kittens, I'd love to have you. Just follow me on Twitch or subscribe on YouTube!
 
-Thank you so so so much for listening! I hope I will be able to get to know you!
+Thank you so so so much for +listening+ (reading?)! I hope I will be able to get to know you!
```
Comment (yayoi-chi) [inline content/en/introduction.org:21]: You commit this one has is. I should probably not use words that refer to spoken language 😅

**Review by yayoi-chi — CHANGES_REQUESTED** [review]
Can you look my comment on line 13? thankies~ thankies~

**Review by yayoi-chi — APPROVED** [review]
Wish we had the same version of Hugo. Guess we'll have to live with these types of commits in proofing for now 😅

**Review by yayoi-chi — CHANGES_REQUESTED** [review]
Still need changes from the last commit though 😝

File: content/en/introduction.org (hunk 1 above)
Comment (jjon-saxton) [reply content/en/introduction.org:13]: I like "I have had to find new ways to communicate in order to fulfill my mission", but you would lose the reference about your kittens, so maybe something like:

```suggestion
My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to fulfill my mission, and protect my kittens!
```

**Review by yayoi-chi — APPROVED** [review]

**Pull request merged**